zip-1014.rst

```diff
@@ -35,6 +37,9 @@ as described in sections 3.9 and 7.7 of the Zcash Protocol Specification.
 "Electric Coin Company", also called "ECC", refers to the Zerocoin Electric
 Coin Company, LLC.
 
+"Bootstrap Project", also called "BP", refers to the 501(c)(3) nonprofit
+corporation of that name.
+
 "Zcash Foundation", also called "ZF", refers to the 501(c)(3) nonprofit
 corporation of that name.
 
```

Comment:
```suggestion

"Section 501(c)(3)" refers to that section of the U.S. Internal Revenue Code.
```

Reply:
Also define "Community Panel". Note that the current reference [#zf-community]_ refers to a snapshot of the Community Advisory Panel as of Q4 2019, whereas we probably want to refer to the panel as it evolves over time.